docs/firmware/device-sdk/firmware-upgrade/guide-dfu.md

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Guide - Firmware Update'

```

Comment:
I'm not a fan of this title structure, ex. "Guide - " & "Concept - ". It adds verbosity.

Let's follow the convention from the Cloud section. Either "Firmware Update Overview" or simply "Overview." The nav hierarchy already provides the context.

I also realize the cloud section is inconsistent and we should clean that up separately.